### PR TITLE
(#1024) (#1016) Updates USA-Section in Example Pages and Storybook

### LINKED ISSUE #1007: Remove utility classes from nci-hero markup.
Remove all utility classes in nci-hero. Be sure to check:
- [x] gatsby documentation site
- [x] ncids-js-testing example site
- [x] ncids-css-testing storybook 
 
This util dependency is unnecessary and the hero can be used without the classes stinking up the place.

### LINKED ISSUE #1013: Remove `usa-section--nci-cta-strip` from CTA Strip Examples
There seems to be an extra `usa-section--nci-cta-strip` class added to the CTA Strip examples (including storybook). This should be removed.

Please also remove the `aria-label="Call to action"` on all the examples too and in the documentation site if it is there. The aria-label should not be used if it is going to be "junk".

### LINKED ISSUE #1016: NCI Hero with CTA strip should not contain usa-section
The `usa-section` class is for handling padding of elements that make up the home/landing (or any type of stacked page) layer cake. Having a usa-section inside the nci-hero on the CTA adds too much spacing to the CTA. Really, the section just controls the padding and background color, so we can easily just target either a wrapper or the CTA itself.  Also, it should probably not even be padding but a margin or something.

This needs to be done at the same time as #1024.

### LINKED ISSUE #1017: NCI Hero with CTA strip heading/tagline should set color
NCI Hero with CTA strip heading/tagline should set color

Color is being overriden to accent-cool. And it should be ____?

Related: #1016

Screenshot 2023-05-30 at 3 49 52 PM
Note: The hero is `usa-section--dark` in this screenshot

### LINKED ISSUE #1024: Update usa-section spacing
This needs to be done at the same time as #1016.

Note from the devs - negative margin + z-index fix

### PR DESCRIPTION
**Pull request by dlescarbeau**
- Closes #1024
- Closes #1016
- Closes #1007 
- Closes #1013 
- Closes #1017 (Non-issue)

### CODE REVIEW

File: testing/ncids-css-testing/stories/components/nci-cta-strip/default.stories.jsx [hunk 1]
```diff
@@ -4,9 +4,9 @@ import css from './nci-cta-strip.scss';
 
 // language=HTML
 const html = `
-<section class="usa-section usa-section--dark usa-section--nci-cta-strip">
+<section class="usa-section usa-section--dark">
```
Comment (dev-rana-publicis) [inline testing/ncids-css-testing/stories/components/nci-cta-strip/default.stories.jsx:7]: CTA Strip standalone and in the hero have been changed to divs

File: packages/ncids-css/packages/nci-hero/internal/_nci-hero.scss [hunk 2]
```diff
@@ -106,3 +106,11 @@
 .nci-hero--with-cta-strip {
 	background: color($theme-color-accent-cool-darker);
 }
+
+.nci-hero__nci-cta-strip-container {
+	background-color: color($theme-color-accent-cool-darker);
```
Comment (dev-rana-publicis) [inline packages/ncids-css/packages/nci-hero/internal/_nci-hero.scss:111]: Refer to the comment about moving the CTA strip in the hero variant. By moving the CTA strip inside of `<div class="nci-hero nci-hero--with-cta-strip">`, you won't need that background-color line. The parent div should dictate the color of the CTA strip

**Review by dev-rana-publicis — CHANGES_REQUESTED** [review]

**Comment by andyvanavery** [comment]
Reviewed and approved by Andy Van Avery on 14JUN2023. Needs Design review.

**Comment by dev-rana-publicis** [comment]
The PR looks good to me after combing through it multiple times. Only aspect that I might add is the `aria-label` on the anchor tags to show that we are using `aria-label` and not `alt` text.

File: testing/ncids-css-testing/stories/templates/cgov-home/default.stories.jsx [hunk 3]
```diff
@@ -255,7 +283,7 @@ const html = `
 							href="https://www.cancer.gov"
 							alt="NCI Equity and Inclusion Program"
```
Comment (dev-rana-publicis) [inline testing/ncids-css-testing/stories/templates/cgov-home/default.stories.jsx:284]: Change `alt` to `aria-label`

**Review by dev-rana-publicis — CHANGES_REQUESTED** [review]

**Review by dev-rana-publicis — APPROVED** [review]

**Review by monika-jaeger — APPROVED** [review]

**Review by arcepaul — APPROVED** [review]

**Pull request merged**